fuse/readonly/mount_unix.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 // Mount mounts ipfs at a given location, and returns a mount.Mount instance.
-func Mount(ipfs *core.IpfsNode, mountpoint string) (mount.Mount, error) {
+func Mount(ipfs *core.IpfsNode, mountpoint string, allow_other bool) (mount.Mount, error) {
```

Comment:
we have the `node` here, so let's just pull out the config here:

```go
cfg := node.Repo.Config()
allow_other := cfg.Mounts.FuseAllowOther
```
and thus avoid changing the interfaces